scripts/zloop.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
Is there a need to convert this from bash to sh?

Reply:
Not if changing the shebang here is acceptable. It should be automatically mangled to an appropriate absolute path by packaging scripts for the installed version of the script.

Reply:
Sorry, I meant more specifically should we removing any bash'isms from this script.

Reply:
The bashisms are fine when the shell is bash, it's more an issue when /bin/sh is assumed to be bash or when the path to bash is assumed to be /bin/bash in a script that could trivially be using /bin/sh. In this script there are a number of bashisms that justify using bash. We have the package installed, might as well use it.